### ISSUE TITLE
[Frontend] Same view on page refresh

### ISSUE THREAD

**Issue by trondoe** [Puzzlepart/did#415]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
When doing a page refresh (F5) the same view should be displayed afterwards.

### Description
When I'm at the Sammendrag view and have made a change in my calendar and want to refresh the Sammendrag-view by performing a page refresh (F5) in the browser, the view changes to Oversikt. See screenshots below. When performing a page refresh, it makes sense to keep the active view after the refresh.

Before page refresh:
![image](https://user-images.githubusercontent.com/8970087/82992017-d4dd3d80-9ffe-11ea-8c8a-109d0353e532.png)

After page resfresh:
![image](https://user-images.githubusercontent.com/8970087/82992077-ec1c2b00-9ffe-11ea-85cd-c69043ea0925.png)
